src/material/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -168,7 +168,7 @@ const _MatTabLinkMixinBase:
   inputs: ['disabled', 'disableRipple', 'tabIndex'],
   host: {
     'class': 'mat-tab-link',
-    '[attr.aria-current]': 'active',
+    '[attr.aria-current]': 'active ? "page" : null',
```

Comment:
Should we have a test for it?

Reply:
Yeah, I sent the PR before updating the test to grab the number